components/layouts/content-list/content-list--years.hbs

```diff
@@ -0,0 +1,79 @@
+<div class="section--bg-prim-blue">
+  <div class="content-list section__inner">
+    <h1 class="content-list__headline">Where we're at today.</h1>
+
+    <div class="content-years__preface themed-text">
+      <p>Because we don’t manufacture any goods, our carbon footprint is largely
+        driven by office utilities, employee commuting, business travel, and
+        technology systems.</p>
+
+      <p>Taking these factors into account and using the
+        <a
+          href="https://www.epa.gov/climateleadership/scope-1-and-scope-2-inventory-guidance"
+        >EPA guidelines for calculating</a>
+        carbon generation, we’ve calculated our company’s global carbon
+        footprint (including internal staff and talent) for 2019, 2020, and 2021
+        (projected):</p>
+    </div>
+
+    <dl class="content-years">
```

Comment:
Can we change this to a less datatype-specific name? The first time we're asked to use this patter with something that isn't a year, the markup won't make sense. How about something like `stat-detail` - stat on the left, whatever it is, and the details for that stat on the right.

Reply:
yas I like it. The name I gave it made me feel icky.